Day6_excercise.rb

```diff
@@ -0,0 +1,77 @@
+#1.store names of students
+names = ["Jessie", "Wendy", "Kelly"]
+#2.store students' names and their ages
+students = {Jessie: 29, Wendy: 22, Kelly: 33}
+#3. store students' names, ages and their favorit colors.
+students = {
```

Comment:
Don't know if the question asks, but do you need to be able to handle the case where there are duplicate names?

Reply:
it can store duplicate names

Reply:
Nope, with the hash structure, you can't store duplicate keys.

Reply:
this is just an example that textbook ask to do, not a question